test/absform_tests.erl

```diff
@@ -7,14 +7,13 @@ function_type_list_to_fun_types_test() ->
         merl:quote("-spec f(T)-> boolean() when T :: tuple();"
                    "       (atom()) -> any()."),
     BoundedFunTypeList = absform:normalize_function_type_list(FunTypeList),
-    {Ty, Cs} = absform:function_type_list_to_fun_types(BoundedFunTypeList),
-    ?assertMatch({type, 1, union,
+    Ty = typechecker:bounded_type_list_to_type({tenv, [], []}, BoundedFunTypeList),
+    ?debugFmt("Ty = ~p\n", [Ty]),
```

Comment:
Delete debug stuff

Reply:
Oops. Incidentally, is there a way to make eunit actually print the mismatching term (or is there a log file to look at). Currently it gives me something like
```
Pattern {bla, bla, [big, term]} doesn't match {bla, bla, ...}
```
(with literal `...`) which isn't very helpful.

Reply:
Once I made a function for computing the diff of terms and used it in tests like this.

Reply:
I'd be happy to do the diff manually, if only it would print the full term.